examples/fluentd/nginx/README.md

```diff
@@ -0,0 +1,128 @@
+# Application Performance Monitoring with NGINX
+
+This example extends Rick Nelson's post [Using NGINX Logging for Application Performance Monitoring](https://www.nginx.com/blog/using-nginx-logging-for-application-performance-monitoring/) by describing how to feed log data from NGINX to Beacon via Fluentd.
+
+## Pre-requisites
+
+This post assumes one or more NGINX-hosted applications already exist and Fluentd is already running and has access to the NGINX logs.  If you're looking for guidance around installation and setup, refer to the documentation for [NGINX](https://www.nginx.com/) and [Fluentd](https://docs.fluentd.org).
+
+## Update NGINX logging configuration
+
+The below is an example log format containing some NGINX timing variables.  We use JSON formatting to enable consumption via Fluentd without the use of additional parsing plugins.
+
+```
+log_format apm escape=json
+    '{'
+        '"client":"$remote_addr",'
+        '"request":"$request",'
+        '"request_length":$request_length,'
+        '"bytes_sent":$bytes_sent,'
+        '"body_bytes_sent":$body_bytes_sent,'
+        '"referer":"$http_referer",'
+        '"user_agent":"$http_user_agent",'
+        '"upstream_addr":"$upstream_addr",'
+        '"upstream_status":$upstream_status,'
+        '"request_time":$request_time,'
+        '"upstream_response_time":$upstream_response_time,'
+        '"upstream_connect_time":$upstream_connect_time,'
+        '"upstream_header_time":$upstream_header_time'
+    '}';
+
+access_log /var/log/nginx/apm.log apm;
+```
+
+Once enabled, JSON should be visible in the log for processed requests:
+
+```
+# tail /var/log/nginx/apm.log
```

Comment:
nit: maybe pipe to jq for easy reading.    `tail /var/log/nginx/apm.log | jq`

Reply:
Agree `jq` makes it more readable.  In many cases though, I think jq is unlikely to be installed.  For that reason, I'm hesitant to add it.